jquery.are-you-sure.js

```diff
@@ -78,15 +79,21 @@
         return (getValue($field) != origValue);
       };
 
+
+
       var $form = ($(this).is('form')) 
                     ? $(this)
                     : $(this).parents('form');
 
       // Test on the target first as it's the most likely to be dirty
       if (isFieldDirty($(evt.target))) {
+        setDirtyFieldStatus($(evt.target), true);
```

Comment:
I think it's better to set this in the "brute force" section, I noticed that rescans won't pickup on this section.